posthog/session_recordings/sql/session_replay_event_migrations_sql.py

```diff
@@ -114,3 +114,49 @@
     table_name=SESSION_REPLAY_EVENTS_DATA_TABLE(),
     cluster=settings.CLICKHOUSE_CLUSTER,
 )
+
+ALTER_SESSION_REPLAY_DROP_SOURCE_COLUMN = """
+    ALTER TABLE {table_name} on CLUSTER '{cluster}'
+    DROP COLUMN IF EXISTS snapshot_source
+"""
+
+DROP_SOURCE_DISTRIBUTED_SESSION_REPLAY_EVENTS_TABLE_SQL = lambda: ALTER_SESSION_REPLAY_DROP_SOURCE_COLUMN.format(
```

Comment:
I really don't like the name of this `DROP_SOURCE_DISTRIBUTED_SESSION_REPLAY_EVENTS_TABLE_SQL` It reads like a drop table and doesn't reference that it's just a column. When I first looked at the migration above I was worried that we were dropping the whole table

Reply:
no, you were copying and pasting previous blocks and using find and replace 🙈